backoffice/views.py

```diff
@@ -305,3 +305,32 @@ def vehicles(request):
     context["page_range"] = page_range
 
     return render(request, "backoffice/vehicles.html", context)
+
+
+@require_backoffice_access
+def close_booking(request, booking_id, should_lock=True):
+    context = {
+        "menu": "bookings",
+        "user": request.user,
+    }
+
```

Comment:
There needs to be some validation here. 

We need to check the current state of the booking is one of `STATE_ACTIVE`, `STATE_PENDING` or `STATE_LATE` - from any other states it's not valid to transition to `STATE_INACTIVE`. 

As an aside: it could actually be a race condition in theory even so - we ought to really do an optimistic lock on the row (i.e. the SQL query only updates the booking state if and only if the state going into the transaction is one of those three predecessor states), but I'm not yet sure how to do this in Django (there's a few other places we need it once we figure out how). However, if you don't feel like that as part of this PR I'm happy to take it with just a basic conditional check here and a FIXME noting a possible race condition between the check and the `booking.save()` call.

We should also only update the box current booking to `None` if box.current_booking is the same as the booking we are trying to close. Once again, this could be racy, the same approach to a fix as above would work, but also could just do a FIXME.

Reply:
There might be other sanity checks needed - those two just immediately sprung to mind. Might be worth thinking through "when would we not want to make these changes" to see if we need any more.